### DISCUSSION TITLE
Couple of questions

### DISCUSSION THREAD

**Discussion by glenelkinsdev** [crazy-max/swarm-cronjob#254] [Q&A]
So far i'm loving this, it's the only swarm scheduler i have found that actually works. I do have a couple of questions:

1) When my service is running in replicated mode, lets say i say 9 replicas, it seems to automatically go back to just 1. Is this the correct behaviour? I ask because i have a container that processes video files, converts the format etc i don't want it to just run 1 container every minute, i want it to be able to use the swarm to process say 10 videos at once, thus replicating it across my nodes. Is there a way to do this?

2) Is there a way to make it run every 20 seconds?

**Comment by dberardo-com** [comment]
i would add to this another important question: i have noticed that when a container is restarted, then the cronjob is rescheduled automatically, regardless of the actual cron scheulde rule. This is really not desirable. is there a way to prevent this?

**Comment by ivan-kleshnin** [comment]
It's not supposed to work with `$ docker compose up`, right? 

I guess `deploy.labels` are applicable only to Swarm mode, but still I'm interested what it would take to be able to run a cronjob for a basic compose.
